lib/template_data/intro.md

```diff
@@ -1,11 +1,116 @@
 # `PySAL` Notebooks Book
 
-This is a compilation of official notebooks demonstrating the functionality of
-`PySAL`, the Python Spatial Analysis library.
+This is a compilation of official notebooks demonstrating the functionality of `PySAL`, the
+Python Spatial Analysis library.
 
----
+* * *
 
-**IMPORTANT**: fill in here with more general information about PySAL and the 
-notebooks project
+# Introduction
 
----
+The PySAL project began began life a decade ago as a collaboration between Serge Rey and
+Luc Anslin as a way to coalesce their research efforts on spatial econometrics and
+spatio-temporal dynamics.
+Over the last 10 years, the library has grown considerably, welcoming new developers to
+the team, supporting several doctoral dissertations, and including functions to facilitate
+a much broader set of spatial data science applications than envisioned originally at the
+outset. For that reason, PySAL has taken a new form and grown into a new modular structure
+as it has matured, permitting greater flexibility for both users and developers.
+Unfortunately, that flexibility comes at a cost, since it can be difficult to discover and
+understand all the capabilities that exists within the PySAL ecosystem, and for that
+reason we’ve decided to collect a wide variety of example notebooks, drawn from each of
+the modules in the PySAL ecosystem, and combined them into this jupyterbook.
+We hope this decision will help elucidate everything PySAL can do—and maybe even inspire
+some new developments we have yet to discover.
+This is an ongoing project, and it will evolve both on its own, and with the constituent
+packages and notebooks that form its substance.
+We hope you will find it useful.
+
+# PySAL
+
+The original design of PySAL was to have a single monolithic library with subcomponents
```

Comment:
Given the notebooks project will hopefully be visited by a lot of newcomers who want to do *something* with PySAL, it might be good to start upfront with how the library is structured and what its components are. If so, I'd suggest to move these two paragraphs to the bottom and re-title them as "The PySAL history" or something to that effect.

Reply:
cool. I'd wondered about this as i was writing... this history is an important part of the story of why we're doing this book, but also dont want to scare folks away with too much background before getting to the fun stuff